src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTestData.java

```diff
@@ -50,6 +50,20 @@ public class JSONCodecRegistryTestData {
   public final String STRING_WITH_2BYTE_UTF8_CHAR = "text-with-2-byte-utf8-\u00a2"; // cent symbol
   public final String STRING_WITH_3BYTE_UTF8_CHAR = "text-with-3-byte-utf8-\u20ac"; // euro symbol
 
+  // Date/time test values
+  public final String DATE_VALID_STR = "2024-09-24";
+  public final String DATE_INVALID_STR = "not-a-date";
+
+  public final String DURATION_VALID1_STR = "1h30m";
+  public final String DURATION_VALID2_STR = "PT1H30M";
+  public final String DURATION_INVALID_STR = "not-a-duration";
+
+  public final String TIME_VALID_STR = "12:34:56.789";
+  public final String TIME_INVALID_STR = "not-a-time";
+
+  public final String TIMESTAMP_VALID_STR = "2024-09-12T10:15:30Z";
```

Comment:
Do timestamps support milliseconds, like `"2024-09-12T10:15:30.123Z"` ?

Reply:
Yes, they should (and even micro-, nano-seconds I think).
It'd great if you could check on client side (and obv. should add more tests on server side).

Reply:
Milliseconds yes, `2024-09-30T21:30:08.212Z` seems to be stored correctly, but micro and nano doesn't seem to work, `2024-09-30T21:30:08.212123456Z` saves without error but I just get back `2024-09-30T21:30:08.212Z`

Reply:
Ah. Now that I think, I suspect this is Cassandra limitation, unfortunately. I think it stores values as 64-bit milliseconds-since-epoch (1970-01-01/UTC).

So yeah, we can accept more granular but not retain.

Reply:
Might be good to throw an error in the case of microseconds? Right now the additional decimal places are silently dropped.